### PR TITLE
Improve API docs

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2287

## Examples
N/A

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Client.bal [hunk 1]
```diff
@@ -50,72 +49,65 @@ public isolated client class Client {
...
     returns sql:ExecutionResult|sql:Error = @java:Method {
         'class: "io.ballerina.stdlib.mssql.nativeimpl.ExecuteProcessorUtils",
         name: "nativeExecute"
     } external;
 
-    # Executes a provided batch of parameterized DDL or DML SQL queries
-    # and returns the summary of the execution.
+    # Executes the SQL query with multiple sets of parameters in a batch. Only the metadata of the execution is
+    # returned (not results from the query). If one of the commands in the batch fails (except syntax error),
+    # the `sql:BatchExecuteError` will be differed until the rest of the commands are completed.
```
Comment (kaneeldias) [inline ballerina/Client.bal:88]:
```suggestion
    # the `sql:BatchExecuteError` will be deferreduntil the rest of the commands are completed.
```

File: ballerina/Client.bal [hunk 2]
```diff
@@ -148,20 +138,15 @@ type ClientConfiguration record {|
     sql:ConnectionPool? connectionPool;
 |};
 
-# MSSQL database options.
+# Provides a set of configuration related to MSSQL database.
```
Comment (kaneeldias) [inline ballerina/Client.bal:141]:
```suggestion
# Provides a set of configurations related to MSSQL database.
```

Comment (niveathika) [reply ballerina/Client.bal:141]: Since it's a set, should it be `a set of configuration`? 🤔

File: ballerina/Client.bal [hunk 3]
```diff
@@ -148,20 +138,15 @@ type ClientConfiguration record {|
...
-#                  failed connection. A zero value indicates that the timeout is the
-#                  default system timeout, which is specified as 15 seconds by default.
-#                  A non-zero value is the number of seconds the driver should wait
-#                  before timing out a failed connection
+# + socketTimeout - Socket timeout (in seconds) during the read/write operations with the MSSQL server
+#                   (0 means no socket timeout)
+# + queryTimeout - Timeout (in seconds) to be used when executing a query.
+#                  (-1/0 means no query timeout)
+# + loginTimeout - Timeout (in seconds) when connecting to the MSSQL server and authentication.
+#                  (0 means 15s of login timeout which is the default behaviour of the driver)
```
Comment (daneshk) [inline ballerina/Client.bal:149]: if the login timeout is configured as 0 by a user, Is it correct to set default as 15s

Comment (niveathika) [reply ballerina/Client.bal:149]: This is actually a bug and needs to be fixed. I was planning to send the fix seperately.

**Review by kaneeldias — DISMISSED** [review]

File: ballerina/Client.bal (hunk 3 above)
Comment (daneshk) [reply ballerina/Client.bal:149]: Ack. Better we do not mention it in API docs. shall we just say? (default timeout is 15s)

Comment (niveathika) [reply ballerina/Client.bal]: Fixed in 46adcef

**Review by kaneeldias — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**